src/ducks/Signals/VisualBacktestChart.js

```diff
@@ -48,19 +52,17 @@ const VisualBacktestChart = ({ data, price, metrics, showAxes = false }) => {
 
         {generateMetricsMarkup(metrics, {
           data: {
-            active_addresses: formattedData
+            active_addresses: triggeredSignals
```

Comment:
I am not sure, Why active_addresses. What is it? It should be only for one type of signals DAA. Actually we hide it yet.) Why not triggeredSignals? @GarageInc @DmitriVanGuard

Reply:
I think that it could be refactored. It looks like bad code with side-effects in data-field.
It come here before my changes, I did't removed it.